src/editor.js

```diff
@@ -1465,6 +1467,14 @@ export class OdooEditor extends EventTarget {
                     const range = selection.getRangeAt(0);
                     setCursor(range.endContainer, range.endOffset);
                 }
+                // check for url after user insert a space
```

Comment:
Check for url after user insert a space so that we don't transform an incomplete url.